plugins/mission/mission.h

```diff
@@ -25,25 +38,97 @@ class Mission
         UNKNOWN
     };
 
+    /**
+     * Returns a human-readable English string for an Mission::Result.
+     */
     static const char *result_str(Result result);
 
+    /**
+     * Callback type for async mission calls.
+     */
     typedef std::function<void(Result)> result_callback_t;
 
-
+    /**
+     * Sends a vector of mission items to the device (asynchronous).
+     *
+     * The mission items are uploaded to a drone and can then be started and executed even if a
```

Comment:
` The mission items are uploaded to a drone. Once uploaded the mission can be started and executed even if a connection is lost.`

Reply:
thx.